src/translations/en.json

```diff
@@ -5837,7 +5837,7 @@
         },
         "refresh_tokens": {
           "header": "Refresh tokens",
-          "description": "Each refresh token represents a login session. Refresh tokens will be automatically removed when you click log out. The following refresh tokens are currently active for your account.",
+          "description": "Each refresh token represents a login session. Refresh tokens will be automatically removed when you click log out. Unused refresh tokens will be automatically removed after 90 days. The following refresh tokens are currently active for your account.",
```

Comment:
Is this very important to mention? As translations will still have the old text until they are updated...

Reply:
I'm not sure, because it is considered as breaking change, it will be mentioned in the release notes, so I think it's not urgent to be translated